config/recipes/beats/README.md

```diff
@@ -0,0 +1,39 @@
+# Beats Configuration Library
```

Comment:
Can "library" be a confusing term to use? To me it sounds more like a programmatic thing than a collection of examples. I'd appreciate thought from better english speakers though (cc. @anyasabo @charith-elastic)

Reply:
Good point! Maybe we should just go with "Beats Configuration Examples"?

Reply:
_The Beats Zoo_?  :stuck_out_tongue: 
+1 for _Beats Configuration Examples_

Reply:
👍 Examples or Cookbook I think both avoid the ambiguity seb mentioned